sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergCatalogConfig.java

```diff
@@ -84,17 +84,26 @@ public org.apache.iceberg.catalog.Catalog catalog() {
   }
 
   public void createTable(
-      String tableIdentifier, Schema tableSchema, @Nullable List<String> partitionFields) {
+      String tableIdentifier,
```

Comment:
This method is used by Beam SQL's `CREATE TABLE ...` command. 

Let's revert these changes for now, we can add integration with Beam SQL later